com.microsoft.mrtk.input/Visualizers/HandJointVisualizer/HandJointVisualizer.cs

```diff
@@ -29,10 +29,10 @@ public class HandJointVisualizer : MonoBehaviour
 
         [SerializeField]
         [Tooltip("Joint visualization mesh.")]
-        private Mesh jointMesh;
+        private UnityEngine.Mesh jointMesh;
```

Comment:
I'm curious why this change is needed?   I assume you're hitting conflicts with you product namespace, but what I don't understand is why.   Unless you're adding a reference to your product assemblies, I don't see how com.microsoft.mrtk.input would pick up your product's namespace.